resources/js/components/CsvTable.vue

```diff
@@ -186,102 +118,113 @@
-                    res.data = state.dsv.parse(content.value);
-                } else {
+                    res.data = state.dsv.parse(props.content);
+                }
+                else {
                     const headerPlaceholder = [];
-                    for(let i=0; i<header.length; i++) {
-                        headerPlaceholder.push(`#${i+1}`);
+                    for(let i = 0; i < header.length; i++) {
+                        headerPlaceholder.push(`#${i + 1}`);
                     }
-                    res.data = state.dsv.parseRows(content.value);
+                    res.data = state.dsv.parseRows(props.content);
                     res.header = headerPlaceholder;
                 }
                 state.computedRows = res;
                 state.computedRows.striped_data = res.data.slice(state.stripedStart, state.stripedEnd);
-
                 if(!internal) {
                     context.emit('parse', state.computedRows);
                 }
             };
 
-            // DATA
-            const state = reactive({
+
+            const { value: csvSettings } = useLocalStorage('csv-settings', {
                 delimiter: ',',
                 hasHeaderRow: true,
                 showLinenumbers: false,
                 showCount: 10,
                 skippedCount: 0,
                 showPreview: true,
+                useCustomDelimiter: false,
+            });
+
+            // DATA
+            const state = reactive({
                 computedRows: {},
-                dsv: computed(_ => d3.dsvFormat(state.delimiter || ',')),
+                dsv: computed(_ => d3.dsvFormat(csvSettings.delimiter || ',')),
+                endVisible: computed(_ => {
+                    return state.stripedStart + csvSettings.showCount > state.rows;
+                }),
                 rows: computed(_ => state.computedRows.data ? state.computedRows.data.length : 0),
-                maxRows: computed(_ => state.rows - state.skippedCount),
-                maxSkippedRows: computed(_ => state.rows > 0 ?  state.rows - 1 : 0),
-                stripedStart: computed(_ => state.skippedCount || 0),
+                maxRows: computed(_ => state.rows - props.skippedCount),
+                maxSkippedRows: computed(_ => state.rows > 0 ? state.rows - 1 : 0),
+                stripedStart: computed(_ => csvSettings.skippedCount || 0),
                 stripedEnd: computed(_ => {
-                    return Math.min(
-                        (state.skippedCount || 0) + (state.showCount || 10),
-                        state.rows
-                    );
+                    return Math.min((csvSettings.skippedCount || 0) + (csvSettings.showCount || 10),
+                        state.rows);
                 }),
             });
 
             onMounted(_ => {
                 recomputeRows();
             });
-            watch(_ => content.value, _ => {
+            watch(_ => props.content.value, _ => {
```

Comment:
is `.value` correct here?

Reply:
Seems like a error: Fixed in bbf61a5c97a99fab5820e987765c064f9b057140